src/pii-client.ts

```diff
@@ -0,0 +1,79 @@
+import PIIsdk, {
+    AgentType,
+    initAgent,
+    PIIEncryptionMethod,
+} from "@notabene/pii-sdk";
+import { TransactionArguments, TravelRuleOptions } from "./types";
+
+export class PIIEncryption {
+    private readonly config: TravelRuleOptions;
+    public toolset: PIIsdk;
+
+    constructor(config: TravelRuleOptions) {
+        this.config = config;
```

Comment:
You can simply set `private readonly` in constructor variable instead of declaring it in line 9 and assigning it in line 13

Reply:
Refactor to set as onstructor(private readonly config: TravelRuleOptions ...